### PR TITLE
description of count field, changes default readme

### PR DESCRIPTION
**Pull request by abdulfaizp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,111 +1,5 @@
...
-Also, thanks to [Sauce Labs](http://saucelabs.com) for helping sponsor the project.
-
-Special Thanks
---------------------
-- [Middleman](https://github.com/middleman/middleman)
-- [jquery.tocify.js](https://github.com/gfranko/jquery.tocify.js)
-- [middleman-syntax](https://github.com/middleman/middleman-syntax)
-- [middleman-gh-pages](https://github.com/edgecase/middleman-gh-pages)
-- [Font Awesome](http://fortawesome.github.io/Font-Awesome/)
+Any and all contributions are welcome! If you spot a tpyo or error, feel free to fix it and submit a pull request :)
```
Comment (xoneco) [inline README.md]: Yas! That's even better

File: source/index.html.md [hunk 2]
```diff
@@ -274,6 +274,7 @@ roomname | `Cruciform Building B.3.05` | The name of the room. It often includes
 siteid | `044` | Every room is inside a site (building). All sites have ids.
 weeknumber | `8` | The week the booking is in.
 phone | `45699` | Phone number (UCL extension)
+count | `1197` | Total number of bookings matching the query. `count` field will only be in the first response to a query
```
Comment (xoneco) [inline source/index.html.md]: *The* count field @emilyruby

Comment (abdulfaizp) [inline source/index.html.md]: why are you tagging her. stop it

**Comment by abdulfaizp** [comment]
squash and merge it please

**Pull request merged**